src/main/java/dk/sdu/se_f22/contentmodule/management/Data/Management.java

```diff
@@ -0,0 +1,118 @@
+package dk.sdu.se_f22.contentmodule.management.Data;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import dk.sdu.se_f22.sharedlibrary.models.Content;
+
+public class Management {
+    static Database dat;
+
+    static {
+        try {
+            dat = Database.getDatabase();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static int Create(String html) throws SQLException {
+        Scanner s;
+
+        try {
+            var res = dat.Execute("INSERT INTO pages (html, timestamp) VALUES ( '" + html + "', NOW()) RETURNING id;");
+            res.next();
+            return res.getInt(1);
+        } catch (Exception e) {
+            e.printStackTrace();
+            StringBuilder scriptString = new StringBuilder();
+            try {
+                s = new Scanner(new FileInputStream("src/main/resources/dk/sdu/se_f22/contentmodule/management/PostgresScript.txt"));
+
+                while (s.hasNext()) {
+                    scriptString.append(s.nextLine());
+                }
+                dat.executeVoidReturn(scriptString.toString());
+            } catch (FileNotFoundException ex) {
+                throw new RuntimeException(ex);
+            }
+       }
+
+        return 0;
+    }
+
+    public static String getPageString(int id) throws SQLException {
+        var result = GetResultSetFromId(id);
+        try {
+            result.next();
+            return result.getString(2);
+        } catch (SQLException e) {;
+            return null;
+        }
+    }
+
+    public static Content[] GetArrayOfContent(int[] ids) {
+        List<Content> contents = new ArrayList<Content>();
+        StringBuilder sb = new StringBuilder();
+        sb.append("SELECT * FROM pages WHERE id IN (");
```

Comment:
This also should ***NEVER BE DONE***! - Please see my other comment   
There is better ways of doing this